torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9542,9 +9535,7 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                # RuntimeError:
                # Arguments for call are not valid.
                DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit', dtypes=(torch.complex64,)),
-               # RuntimeError: aliasOp != torch::jit::getOperatorAliasMap().end()
-               # INTERNAL ASSERT FAILED at "../torch/csrc/jit/passes/utils/check_alias_annotation.cpp":157,
-               # please report a bug to PyTorch.
+               # RuntimeError: Arguments for call are not valid.
```

Comment:
What are the new errors here ? Do you know why this is happening ?

Reply:
Here is the full message:
```
RuntimeError: 
Arguments for call are not valid.
The following variants are available:
  
  aten::norm.Scalar(Tensor self, Scalar p=2) -> (Tensor):
  Expected a value of type 'number' for argument 'p' but instead found type 'str'.
  
  aten::norm.ScalarOpt_dim(Tensor self, Scalar? p, int[1] dim, bool keepdim=False) -> (Tensor):
  Expected a value of type 'Optional[number]' for argument 'p' but instead found type 'str'.
  
  aten::norm.names_ScalarOpt_dim(Tensor self, Scalar? p, str[1] dim, bool keepdim=False) -> (Tensor):
  Expected a value of type 'Optional[number]' for argument 'p' but instead found type 'str'.
  
  aten::norm.ScalarOpt_dim_dtype(Tensor self, Scalar? p, int[1] dim, bool keepdim, *, int dtype) -> (Tensor):
  Expected a value of type 'Optional[number]' for argument 'p' but instead found type 'str'.
  
  aten::norm.dtype_out(Tensor self, Scalar? p, int[1] dim, bool keepdim, *, int dtype, Tensor(a!) out) -> (Tensor(a!)):
  Expected a value of type 'Optional[number]' for argument 'p' but instead found type 'str'.
  
  aten::norm.out(Tensor self, Scalar? p, int[1] dim, bool keepdim=False, *, Tensor(a!) out) -> (Tensor(a!)):
  Expected a value of type 'Optional[number]' for argument 'p' but instead found type 'str'.
  
  aten::norm.ScalarOpt_dtype(Tensor self, Scalar? p, *, int dtype) -> (Tensor):
  Expected a value of type 'Optional[number]' for argument 'p' but instead found type 'str'.
  
  aten::norm.names_ScalarOpt_dim_dtype(Tensor self, Scalar? p, str[1] dim, bool keepdim, *, int dtype) -> (Tensor):
  Expected a value of type 'Optional[number]' for argument 'p' but instead found type 'str'.
  
  aten::norm.names_dtype_out(Tensor self, Scalar? p, str[1] dim, bool keepdim, *, int dtype, Tensor(a!) out) -> (Tensor(a!)):
  Expected a value of type 'Optional[number]' for argument 'p' but instead found type 'str'.
  
  aten::norm.names_out(Tensor self, Scalar? p, str[1] dim, bool keepdim=False, *, Tensor(a!) out) -> (Tensor(a!)):
  Expected a value of type 'Optional[number]' for argument 'p' but instead found type 'str'.

The original call is:
  File "<string>", line 3

def the_method(i0):
    return i0.norm('fro')
           ~~~~~~~ <--- HERE
```

I suspect there is a mismatch between the passed in operator and the TS node.